server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -229,15 +231,13 @@ static TransportVersion def(int id) {
      * Reference to the earliest compatible transport version to this version of the codebase.
      * This should be the transport version used by the highest minor version of the previous major.
      */
-    @UpdateForV9(owner = UpdateForV9.Owner.CORE_INFRA)
-    // This needs to be bumped to the 8.last
-    public static final TransportVersion MINIMUM_COMPATIBLE = V_7_17_0;
+    public static final TransportVersion MINIMUM_COMPATIBLE = BYTE_SIZE_VALUE_ALWAYS_USES_BYTES_1;
```

Comment:
This doesn't need to be precise at this stage, it just needs to be a value that will be collapsed into whatever 8.18.0 uses, at some point.

Reply:
This is currently the most recent version in the 8.x branch - will need to be updated to the final version in the 8.18 branch when 8.18.0 is cut, correct?

Reply:
Actually not, it doesn't have to be the exact version, just _some_ version between that used by 8.17 and 8.18. This is because this will only actually see the versions that are used for the releases, so `version.onOrAfter(MINIMUM_COMPATIBLE)` will match, even if `MINIMUM_COMPATIBLE` is a few id numbers behind the actual version used for 8.18.

Reply:
I'm honoured you chose my transport version, even if it's a dead version never encountered in production because it got reverted.

Reply:
It's what was deployed to serverless at the time - so the serverless tests pass